api/swagger-spec/oapi-v1.json

```diff
@@ -17700,6 +17700,10 @@
      "customStrategy": {
       "$ref": "v1.CustomBuildStrategy",
       "description": "CustomStrategy holds the parameters to the Custom build strategy"
+     },
+     "jenkinsPipelineStrategy": {
```

Comment:
Jenkins plugin now updates build statuses properly with start time & duration. All looks good in the OpenShift console & cmd line.